### DISCUSSION TITLE
Always getting DOKAN_START_ERROR

### DISCUSSION THREAD

**Discussion by uhonermann** [dokan-dev/dokany#1182] [Q&A]
Hello,

I have a system here where I always get DOKAN_START_ERROR on mount.
This is reproducable with the memfs sample (memfs.exe /l K: /d /e).
There's no other output than this error, the only other thing I can see is some output in the eventlog:

20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Mount entry found: \DosDevices\R: -> \Device\Volume{d6cc17c5-1734-4085-bce7-964f1e9f5de9}  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Found entry with matching mount point.  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Finding mount entry; lockGlobal = 1; mount point = \DosDevices\R:.  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Finished event start with status 3 and flags: 11  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Returning actual mount point R  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Mount point was still not assigned after forcing.  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: IOCTL_MOUNTMGR_CREATE_POINT succeeded.  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: -964f1e9f5de9}".  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Creating mount point: "\DosDevices\R:" for device name: "\Device\Volume{d6cc17c5-1734-4085-bce7  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Warning: mount point creation is being forced.  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Event start using mount ID: 1; device name: \Volume{d6cc17c5-1734-4085-bce7-964f1e9f5de9}.  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Inserted new mount entry.  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: ume{d6cc17c5-1734-4085-bce7-964f1e9f5de9}" created  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: SymbolicLink: "\DosDevices\Global\Volume{d6cc17c5-1734-4085-bce7-964f1e9f5de9}" -> "\Device\Vol  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: ; type: 8  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: \DosDevices\Global\Volume{d6cc17c5-1734-4085-bce7-964f1e9f5de9}"; mount point: "\DosDevices\R:"  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: disk device name: "\Device\Volume{d6cc17c5-1734-4085-bce7-964f1e9f5de9}"; symbolic link name: "  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Creating disk device; mount point = R; mount ID = 1l  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: No mount entry found.  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Finding mount entry; lockGlobal = 0; mount point = \DosDevices\R:.  
20231013 18:31:43 ID:00001 (0x40060001)  VWDGF  -  INFO   -  Src: dokan2  Desc: Entered event start.  

It looks like IoVerifyVolume doesn't trigger DokanMountVolume for some reason, but why ?

Regards,
  Uwe

**Comment by Liryna** [comment]
Hi @uhonermann ,

It is possible that Mount manager auto mount was disabled on the system and thats why the driver letter was not assigned automatically.

Dokan disable it temporarily for mount point folder.
https://github.com/dokan-dev/dokany/blob/b8056db2a1ee64eb7f73ab3d85cd308eaef7e4af/sys/util/mountmgr.c#L254-L280

**Reply by uhonermann** [reply]
Hi Liryna,

thank you, I'll check that.

Regards,
  Uwe

**Reply by uhonermann** [reply]
I checked that, AutoMount is enabled:

![image](https://github.com/dokan-dev/dokany/assets/75378374/eafdb591-6f93-49c9-8a2c-bbb54a5e6c88)

**Reply by Liryna** [reply]
Then there must be a software on the computer blocking the request to reach dokan driver. It could be an antivirus or other of such types of drivers.
